src/routes/Routes.ts

```diff
@@ -0,0 +1,21 @@
+const Routes = {
+  ROOT: '/',
+  LOGIN: '/login',
+  SIGNUP: `/signup`,
+  HOME: '/home',
```

Comment:
```suggestion
  DASHBOARD: '/dashboard',
```

we could rename to anything, but since Home means to the homepage from a website, we should think in a new name to use here.

Reply:
I think we could let the ROOT in the place of the HOME and the logged in Home be called DASHBOARD